### PR TITLE
Added legend view

### PR DESCRIPTION
**Pull request by AmeyaMalekar**
Added the legend for homepage, I'll work on linking the backend filter functions during next sprint.

### CODE REVIEW

File: frontend/src/Legend.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            legend.hidden = false;
+        }
+    }
+
+  return (
+      <div>
+        <button type="button" class="button" id="expand" name="expand" onClick={handleChange}>
+            <ExpandIcon onClick={handleChange}/>
+        </button>
+        <div id="legend-attributes">
```
Comment (ksittner) [inline frontend/src/Legend.js:42]: I think you could condense code and avoid directly calling the DOM API by supplying the expand state variable to the div's hidden attribute in line 42:

<div id="legend-attributes" hidden={expand} >

Comment (AmeyaMalekar) [reply frontend/src/Legend.js:42]: Got it, I'll make the change.

**Comment by eliasow** [comment]
Maybe consider not using braces for single line if/else statements, may make it cleaner

**Comment by davidayu** [comment]
Looks good to me

**Review by davidayu — APPROVED** [review]

**Comment by AmeyaMalekar** [comment]
> Looks good to me

Thanks for the review, David.

**Comment by noahmarxx** [comment]
Looks good!

**Comment by AmeyaMalekar** [comment]
> Looks good!

Thanks Noah.

**Comment by AmeyaMalekar** [comment]
> Maybe consider not using braces for single line if/else statements, may make it cleaner

Noted, thanks Elias.

**Pull request merged**